tests/data/server-content/www.jetbrains.com/pycharm/download/download_thanks.jsp?edition=edu&os=linux

```diff
@@ -0,0 +1,408 @@
+            </li>
+        </ul>
+    </div>
+    <div class="leftfloated block40">
+        <h4>Mobile development</h4>
+        <ul>
+            <li class="objc"><a href="/objc"><i class="icon jb-logo _logo-appcode _grey _size-1"></i>AppCode</a></li>
+            <li class="idea"><a href="/idea#community_edition"><i class="icon jb-logo _logo-idea _grey _size-1"></i>IntelliJ IDEA Community Edition</a></li>
+        </ul>
+        <h4>Education</h4>
+        <ul>
+            <li class="pc-edu"><a href="/pycharm-educational"><i class="icon jb-logo _logo-pycharm _grey _size-1"></i>PyCharm Educational Edition</a></li>
+            <li class="rightfloated block40"><br/><br/><a class="view-all-products" href="/products.html">View all products &raquo;</a></li>
+        </ul>
+    </div>
+    <br class="clr"/>
+</div></div>
+<div id="product">
+<div class="container">
+        <a class="header-logo jb-logo _logo-pycharm _size-header-logo _inverted" href="/pycharm-educational/">PyCharm Educational Edition</a>
+    <span class="nav-logo jb-logo _logo-pycharm _size-3 _inverted"></span>
+
+
+
+
+</div>
+</div>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+<!--start_noindex-->
+<!-- @test -->
+
+<nav>
+    <div class="container">
+        <ul class="nav">
+
+            <li class="pycharm-educational"><a href="/pycharm-educational/" title="Overview">Overview</a></li>
+
+            <li class="concepts"><a href="/pycharm-educational/concepts/" title="How It Works">How It Works</a></li>
+
+            <li class="quickstart"><a href="/pycharm-educational/quickstart/" title="Quickstart Guide">Quickstart Guide</a></li>
+
+            <li class="contest"><a href="/pycharm-educational/contest/" title="Contest">Contest</a></li>
+
+            <li class="current"><a href="/pycharm-educational/download/" title="Download">Download</a></li>
+
+        </ul>
+    </div>
+</nav>
+
+
+<!--end_noindex-->
+
+
+
+
+
+
+<div id="main">
+    <div class="container">
+                <div id="content">
+                            <script type="text/javascript">
+                    jQuery(document).ready(function() {
+                        jQuery.get( "/inc/promo_country.jsp", {pathURI: '/pycharm-educational/download'} , function( data ) {
+                            jQuery( "#country_promo_block_storage" ).html( data );
+                        });
+                    });
+                </script>
+                <div id="country_promo_block_storage" style="display: none"></div>
+
+
+        <h2>Thank you for downloading PyCharm Educational Edition!</h2>
+        <h4 class="sans_serif_normal">Your download should start shortly. If it doesn't, please use one of these direct links: <a href="http://download.jetbrains.com/python/pycharm-educational-fake.tar.gz">HTTP</a> or <a href="https://download.jetbrains.com/python/pycharm-educational-1.0.1.tar.gz">HTTPS</a>.</h4>

```

Comment:
You're replaced the wrong link here so the medium test throws an error (404). Use the HTTPS link, not the HTTP.